src/tests/utils/testTypeOrmModule.ts

```diff
@@ -16,17 +16,12 @@ export class TestTypeOrmModule {
     const typeOrmModule = TypeOrmModule.forRoot({
       type: 'postgres',
       host: 'localhost',
-      port: 5434,
-      username: 'seed',
-      password: 'Ge0rgesMoustaki',
-      database: 'seed',
-      entities: [
-        AdminEntity,
-        __dirname + '/../../**/*.entity.{js,ts}', // for use in the library
-        __dirname + '/../../../dist/**/*.entity.{js,ts}', // for use locally in the exampleApp
-        __dirname + '/../../../exampleApp/node_modules/nestjs-admin/dist/**/*.entity.{js,ts}', // for use in CI in the exampleApp
-        ...config.entities,
-      ],
+      port: parseInt(process.env.DB_PORT, 10),
+      username: process.env.DB_USERNAME,
+      password: process.env.DB_PASSWORD,
+      database: process.env.DB_DATABASE,
+      entities: [__dirname + '/dist/**/*.entity.js', AdminUser],
+      migrations: ['dist/migration/*.js'],
```

Comment:
Didn't need to change the entities and migration configurations to connect to different databases?